server/views/taskList/taskListTemplate.pug

```diff
@@ -23,9 +23,12 @@ block content
   else if licenceStatus.decisions.unsuitableResult
     div.error-banner.marginTopMedium
       strong.bold The offender is presumed unsuitable for HDC release
-  else if user.role === 'CA' && licenceStatus.decisions.caRefused
+  else if user.role === 'CA' && licenceStatus.decisions.caRefused && licenceStatus.decisions.refusalReason
     div.error-banner.marginTopMedium
       strong.bold Home detention curfew refused by prison case admin: #{licenceStatus.decisions.refusalReason}
+  else if user.role === 'CA' && licenceStatus.decisions.caRefused && !licenceStatus.decisions.refusalReason
+    div.error-banner.marginTopMedium
+      strong.bold Home detention curfew refused by prison case admin
```

Comment:
this if/else block has become a bit of a monster. 
Could potentially merge the two cases for ca refused in one through using a ternary operator?

 `#{licenceStatus.decisions.refusalReason ? ': ' +  licenceStatus.decisions.refusalReason : ''}`